README.md

```diff
@@ -81,17 +81,61 @@ After you're finished please remove all the comments and instructions!
 <details>
   <summary>Javascript</summary>
   <ul>
-    <li><a href="https://en.wikipedia.org/wiki/Javascript">CSS</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/Javascript">JS</a></li>
   </ul>
 </details>
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+<!-- LIVE DEMO -->
+
+## 🚀 Video presentation Demo <a name="Video-presentation"></a>
+
+- Click here to [see the video](https://www.loom.com/share/7cb39e1c249443f7840056628a918730)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Click here to [View page](#)
+- Click here to [View page](https://matee-safi.github.io/1-Capstone/Home.html)
```

Comment:
[OPTIONAL] You can choose the branch you deploy and change it later as well. I think it would be better if you can deploy the project so we can see and review if you can deploy it properly.